### DISCUSSION TITLE
Combinations display private attribute names instead of public in the URL

### DISCUSSION THREAD

**Discussion by maxldn5** [PrestaShop/PrestaShop#35982] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

On a product page with combinations, when an attribute value is selected, the attribute name+value groups are appended to the URL e.g.:
122-3154-product.html#/10-attr1_value1/11-attr2_value2

Currently we show the attributes' private names. Since this information is visible publicly, we should show the public names instead.

### Expected behavior

Public attribute names are appended to URL

### Steps to reproduce

1. Go on a product page with combinations
2. Select any attribute value
3. Check the attribute-value permutations added to the URL after #/

### PrestaShop version(s) where the bug happened

8.1.5

### PHP version(s) where the bug happened

8.1

**Comment by ChillCode** [comment]
Agree with you, when didn't know this was the behavior of PS we named attributes from providers like "providername_colors"  "providername_sizess"  but merchants of course didn't want to show providers name to customers.

Following you PR:

https://github.com/PrestaShop/PrestaShop/pull/35975

Thanks!

**Reply by maxldn5** [reply]
Hi @ChillCode, thank you for your feedback. I forgot about this PR and let my branch get way out of date. For simplicity I created a new PR with the same change (adapted to the newest changes to Product.php that took place since). The new PR is #36664.
